exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,23 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
```

Comment:
Hi abed. well build conditional code. but you need to be aware here is asking if the number is greater or less than zero. it could be less than 5 and still positive